homeassistant/components/input_number/__init__.py

```diff
@@ -310,7 +310,7 @@ async def async_decrement(self):
         """Decrement value."""
         await self.async_set_value(max(self._current_value - self._step, self._minimum))
 
-    async def async_update_config(self, config: dict) -> None:
+    async def async_update_config(self, config: ConfigType) -> None:
```

Comment:
Can `__init__` and `from_yaml` be adjusted to use `ConfigType`, to match in `input_number`?